rmc/utils/constraint.py

```diff
@@ -277,16 +278,23 @@ def create_filtered_context_ht(overwrite: bool = True) -> None:
         " annotations..."
     )
     # NOTE: Constraint outlier transcripts are not removed
-    ht = process_context_ht(filter_csq=True, csq={MISSENSE, NONSENSE, SYNONYMOUS})
+    csq = {MISSENSE, SYNONYMOUS, READ_THROUGH}.union(NONSENSES)
```

Comment:
should csq be an arg with a default? I don't think we would want to change what we keep in this HT, but this would make it more flexible if we ever did